platform.md

```diff
@@ -276,10 +275,72 @@ All lifecycle phases:
 
+#### `analyzer`
+Usage:
+```
+/cnb/lifecycle/analyzer \
+  [-analyzed <analyzed>] \
+  [-daemon] \ # sets <daemon>
+  [-gid <gid>] \
+  [-group <group>] \
+  [-layers <layers>] \
+  [-log-level <log-level>] \
+  [-previous-image <previous-image> ] \
+  [-run-image <run-image> ] \
+  [-stack-id <stack-id> ] \
+  [-stack <stack> ] \
+  [-tag <tag>...] \
+  [-uid <uid>] \
+  <image>
+```
+
+##### Inputs
+| Input             | Environment Variable  | Default Value            | Description
+|-------------------|-----------------------|--------------------------|----------------------
+| `<analyzed>`      | `CNB_ANALYZED_PATH`   | `<layers>/analyzed.toml` | Path to output analysis metadata (see [`analyzed.toml`](#analyzedtoml-toml)
+| `<daemon>`        | `CNB_USE_DAEMON`      | `false`                  | Analyze image from docker daemon
+| `<gid>`           | `CNB_GROUP_ID`        |                          | Primary GID of the stack `User`
+| `<group>`         | `CNB_GROUP_PATH`      | `<layers>/group.toml`    | Path to group definition (see [`group.toml`](#grouptoml-toml))
+| `<layers>`        | `CNB_LAYERS_DIR`      | `/layers`                | Path to layers directory
+| `<log-level>`     | `CNB_LOG_LEVEL`       | `info`                   | Log Level
+| `<previous-image>`| `CNB_PREVIOUS_IMAGE`  | `<image>`                | Image reference to be analyzed (usually the result of the previous build)
+| `<run-image>`     | `CNB_RUN_IMAGE`       | resolved from <stack>    | Run image reference
+| `<stack>`         | `CNB_STACK_PATH`      | `/cnb/stack.toml`        | Path to stack file (see [`stack.toml`](#stacktoml-toml))
+| `<tag>...`        |                       |                          | Additional tag to apply to exported image
+| `<uid>`           | `CNB_USER_ID`         |                          | UID of the stack `User`
+
+- **If** `<daemon>` is `false`, `<image>` MUST be a valid image reference
+- **If** `<daemon>` is `true`, `<image>` MUST be either a valid image reference or an imageID
+- **If** `<run-image>` is not provided by the platform the value will be resolved from the contents of stack
+- The lifecycle MUST accept valid references to non-existent images without error.
+- The lifecycle MUST ensure that the `<run-image>` is compatible with the `<previous-image>`. To be compatible, the following conditions must be met:
+  - Stack IDs must match
+  - Platform/Architecture must be the same
+  - `<run-image>` `mixins` must be a superset of `<previous-image>` mixins
+- The lifecycle MUST ensure registry read/write access to `<run-image>`, `<previous-image>`, `<tag>`
+
+##### Outputs
+| Output             | Description
+|--------------------|----------------------------------------------
+| [exit status]      | (see Exit Code table below for values)
+| `/dev/stdout`      | Logs (info)
+| `/dev/stderr`      | Logs (warnings, errors)
+| `<analyzed>`       | Analysis metadata (see [`analyzed.toml`](#analyzedtoml-toml)
```

Comment:
Now that we are writing `run-image` metadata in this output. `analyzed.toml` needs to be an input to `exporter` and `stack.toml` can be removed as an exporter input.

Reply:
`<analyzed>` is already an input, I removed `<stack>` in 6480d2e975ba81d5b61d681f0177539408c834b8